### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by darthaud**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,9 +12,11 @@ Basically this plugin launches EsLint and collects its results into SonarQube. I
```
Comment (handrus) [inline README.md:9]: Link to download the plugin on line 6 is wrong (forked repo).
Also we should update the readme explaining why we forked it, what are we doing different and if we have intention to make PRs to the original plugin.

**Review by handrus — CHANGES_REQUESTED** [review]
@darthaud please fix wrong link and upgrade the first part of the file according to the comment

**Review by handrus — COMMENTED** [review]
[LGTM](http://www.lgtm.in/i/ker1)

[:+1:](http://www.lgtm.in/u/ker1) [:-1:](http://www.lgtm.in/r/ker1)

**Review by handrus — APPROVED** [review]

**Review by FabricioFFC — APPROVED** [review]

**Comment by sleroy** [comment]
Good work guys. I have never seen the notifications about your work before. I have done the merge and I preparing a new version inlcuding your modifications.

Thanks a lot !

**Comment by handrus** [comment]
Hi @sleroy it seems to me that all we have here is already integrated at your repo. Thanks a lot!

**Pull request merged**